src/cli.rs

```diff
@@ -55,6 +55,10 @@ pub struct HookCmd {
     #[arg(long)]
     pub shellpid: Option<u32>,
 
+    /// Clobber overridden environment variables.
+    #[arg(long)]
+    pub clobber: bool,
```

Comment:
i'll admit i wasn't immediately clear that force wouldn't clobber- the interaction between the two seems odd. not sure how much it matters but thought i'd call that out - might mean it'd be valuable to make both names more specific.

Reply:
me too tbh. i think `--force`  not-ideally named. i'll try to think of some alternatives and keep `--force` as a hidden fallback